packages/node_modules/@webex/plugin-meetings/src/member/util.js

```diff
@@ -126,8 +126,27 @@ MemberUtil.isMuted = (status, accessor) => {
   return null;
 };
 
+/**
+ * utility method for getting the recording member for later comparison
+ * @param {Object} controls
+ * @returns {String}
+ */
+MemberUtil.getRecordingMember = (controls) => {
+  if (!controls) {
+    return null;
+  }
+  if (controls.record && controls.record.recording) {
```

Comment:
You could do this instead of the additional if on 139
```suggestion
  if (controls.record && controls.record.recording && controls.record.meta) {
```

Reply:
done